cmd/soroban-cli/src/config/network.rs

```diff
@@ -206,3 +188,33 @@ impl From<&(&str, &str)> for Network {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[tokio::test]
+    async fn test_helper_url_local_network() {
+        let network = Network {
+            rpc_url: "http://localhost:8000".to_string(),
+            network_passphrase: passphrase::LOCAL.to_string(),
+        };
+
+        let result = network
+            .helper_url("GBZXN7PIRZGNMHGA7MUUUF4GWPY5AYPV6LY4UV2GL6VJGIQRXFDNMADI")
+            .await;
+
+        assert!(result.is_ok());
+        let url = result.unwrap();
+        assert_eq!(url.as_str(), "http://localhost:8000/friendbot?addr=GBZXN7PIRZGNMHGA7MUUUF4GWPY5AYPV6LY4UV2GL6VJGIQRXFDNMADI");
+    }
+
+    #[tokio::test]
+    async fn test_helper_url_test_network() {
+        // It might be a bit difficult to conduct client mock here.
+        let friendbot_url = "https://friendbot.stellar.org/secret_key";
+        let addr = "GBZXN7PIRZGNMHGA7MUUUF4GWPY5AYPV6LY4UV2GL6VJGIQRXFDNMADI";
+        let url = Url::from_str(&format!("{friendbot_url}?addr={addr}")).unwrap();
+        assert_eq!(url.as_str(), "https://friendbot.stellar.org/secret_key?addr=GBZXN7PIRZGNMHGA7MUUUF4GWPY5AYPV6LY4UV2GL6VJGIQRXFDNMADI");
+    }
```

Comment:
Because the mock client here is a bit complex, I only verified that `let url = Url::from_str(&format!("{friendbot_url}?addr={addr}")).unwrap();` works as expected. This line of code is copied from the codebase. If you would like me to remove it, please let me know.

Reply:
Added mock testing.